nodewatcher/modules/devices/ubnt/unifi.py

```diff
@@ -0,0 +1,157 @@
+from django.utils.translation import ugettext_lazy as _
+
+from nodewatcher.core.generator.cgm import base as cgm_base, protocols as cgm_protocols, devices as cgm_devices
+
+
+class UbiquitiAPACLite(cgm_devices.DeviceBase):
```

Comment:
This does not seem to follow the naming convention, change `Ubiquiti` to `UBNT`.